dist/debian/build_deb.sh

```diff
@@ -98,6 +98,8 @@ fi
 rm -rf "$BUILDDIR"
 mkdir -p "$BUILDDIR"/scylla-machine-image
 
+git config --global --add safe.directory $(realpath $(dirname $0)"/../..")
```

Comment:
AFAIK this will break packing building using `dbuild` please verify it before

Reply:
It's preposterous to insert global configuration commands into a build script. A build script should only touch its build directly, nothing else.

Reply:
Also, what does git configuration have to do with disabling fstrim? Please practice patch hygiene.

Reply:
removed this commit, and I'll try fixing it in a different way